lib/schema.js

```diff
@@ -108,7 +108,7 @@ function Schema(obj, options) {
   this._indexes = [];
   this.methods = {};
   this.methodOptions = {};
-  this.statics = {};
+  this.statics = options?.statics || {};
```

Comment:
We can unfortunately not use optional chaining yet. 

```suggestion
  this.statics = options && options.statics || {};
```